Dependencies/xr/Include/XR.h

```diff
@@ -301,6 +301,8 @@ namespace xr
                 std::vector<InputSource>& InputSources;
                 std::vector<FeaturePoint>& FeaturePointCloud;
 
+                std::pair<bool,Space>& EyeTrackerSpace;
```

Comment:
>EyeTrackerSpace [](http://example.com/codeflow?start=39&length=15)

should this be a struct compared to a pair? are there other properties that could eventually be added here that won't allow for it to be a std::pair? it feels like .first and .second accessors for this bool and space is worse for readability compared to a struct with field names pertaining to whatever these values are for